src/main/java/Duke.java

```diff
@@ -1,10 +1,85 @@
+import java.util.Scanner;
+
 public class Duke {
```

Comment:
Perhaps you can consider adding some comments to describe each method? The format can be found here under 'Comments': https://se-education.org/guides/conventions/java/basic.html